network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -404,3 +405,45 @@ class GeneralProductPage(ProductPage):
 
     class Meta:
         verbose_name = "General Product Page"
+
+
+class BuyersGuidePage(FoundationMetadataPageMixin, Page):
+    max_count = 1
+    template = 'buyersguide/home.html'
+    subpage_types = [SoftwareProductPage, GeneralProductPage]
+
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        if request.user.is_authenticated:
+            products = ProductPage.objects.all()
+        else:
+            products = ProductPage.objects.live()
```

Comment:
As long as our template keeps the `is_current` filtering. Otherwise we may want to filter them here, and not check for `is_current` in the templates.

Reply:
Good catch, I didnt add that into ProductPage yet. Will add that in shortly, thank you for spotting that and bringing it to my attention!